x-pack/plugins/maps/public/actions/map_actions.ts

```diff
@@ -369,8 +369,10 @@ export function addNewFeatureToIndex(geometry: Geometry | Position[]) {
     }
 
     try {
+      await dispatch(updateEditShape(DRAW_SHAPE.INSERT));
```

Comment:
await is not needed for updateEditShape action since its not async. Please remove await here and in other places where updateEditShape added